x-pack/test/fleet_api_integration/apis/uninstall_token/get.ts

```diff
@@ -488,21 +488,6 @@ export default function (providerContext: FtrProviderContext) {
           expect(body.items).to.eql([]);
         });
 
-        it('should return nothing if searched for managed policy name', async () => {
-          const response = await supertest
-            .get(uninstallTokensRouteService.getListPath())
-            .query({
-              search: generatedManagedPolicyArray[0].name,
-            })
-            .expect(200);
-
-          const body: GetUninstallTokensMetadataResponse = response.body;
-          expect(body.total).to.equal(0);
-          expect(body.page).to.equal(1);
-          expect(body.perPage).to.equal(20);
-          expect(body.items).to.eql([]);
-        });
-
```

Comment:
Duplicate of https://github.com/elastic/kibana/blob/ffe6cf4de75d412dbd1d501e01cd8cae05597bfa/x-pack/test/fleet_api_integration/apis/uninstall_token/get.ts#L476-L489 in the same file